lib/takeaway.rb

```diff
@@ -0,0 +1,95 @@
+require_relative 'customer'
+require_relative 'menu'
+
+class Takeaway
+  attr_reader :order
+
+  def initialize(menu = Menu, customer = Customer)
+    @menu = menu.new
+    @customer = customer.new
+    @order = []
+    @total = 0
+  end
+
+  def see_dishes
+    @menu.print_menu
+  end
+
+  def place_order
+    item = what_would_you_like?
+    quantity = how_many?
+    price = @menu.find_price(item)
+
+    add_to_order(item, quantity, price)
+    update_total(price, quantity)
+
+    place_order if anything_else?
+  end
+
+  def verify_order
+    print_order
+    return reset_order if order_wrong?
+
+    text_confirmation
+  end
+
+  def print_order
+    @order.each do |item|
+      puts "#{item[:item]}(#{item[:quantity]}) ~ £#{item[:price]}"
+    end
+    print_total
+  end
+
+  def text_confirmation
+    @customer.input_number
+    @customer.send_confirmation
+  end
+
+  def reset_order
+    @order = []
+    return "Your order has been reset"
+  end
+
+  private
+
+  def order_wrong?
+    puts "Is everything correct? Y or N"
+    answer = gets.chomp.upcase
+    return false if answer == "Y"
```

Comment:
I think you could have done this in one line:
!(answer == "Y")
or even
!(gets.chomp.upcase == "Y")

Reply:
Yes! Thank you for this! I knew I could do it in one line, but I was heavily struggling with how implement it